packages/wallet/test/integration/cip30mapping.test.ts

```diff
@@ -546,6 +547,25 @@ describe('cip30', () => {
           await expect(api.submitTx(hexTx)).rejects.toThrowError(TxSendError);
         });
       });
+
+      describe('api.getPubDRepKey', () => {
+        test("returns the DRep key derived from the wallet's public key", async () => {
+          const cip95PubDRepKey = await api.getPubDRepKey();
+          expect(cip95PubDRepKey).toEqual(await wallet.getPubDRepKey());
+        });
+        test('throws an ApiError on unexpected error', async () => {
+          (wallet.getPubDRepKey as jest.Mock).mockRejectedValueOnce(new Error('unexpected error'));
+          try {
+            await api.getPubDRepKey();
+            // fail the test if it does not throw
+            expect(false).toBe(true);
```

Comment:
An alternative way to do this that yields a better message is `expect.assertions(3);`

Reply:
Didn't know you can use `expect.assertions(n)` after the assertions :sweat_smile: